locale/zh-cn/download/current.md

```diff
@@ -11,23 +11,23 @@ downloads:
     display-hint: Display downloads for
     intro: >
         在你的平台上下载 Node.js 源码或预编译安装包，然后即可马上进行开发。
-    currentVersion: 当前版本
+    currentVersion: 当前最新版本
```

Comment:
`当前`和`最新`意思相近，`current` 和`当前`比较匹配，`最新`和`latest`更配

Reply:
两者区分，一个是当前最新版本，一个是当前 LTS（长期支持）版本。最新是一种强调，与英文原版保持一致。

Reply:
The level of `a good translation` is something of clear, fully-understood meaning with fewer words in different kinds of cultures.  So before translations we should have a deep understanding of them and try to translate them one by one according to proper nature of understanding:)

For example：The `LTS`  stands for `Long Term Support` (For details, see: https://en.wikipedia.org/wiki/Long-term_support). And the `Current` is beta-like version with latest new features as experiments (Maybe they will be removed or changed later). And this `implicit` meaning is easily taken by foreigners but not easily by Chinese people (For 1st sight, they don't know what `LTS` is and cannot tell the difference between `LTS` and `Current`. Most of them may download the latter).

So all in all, I perfer to translate them as:

1）`LTS`：正式版  ( A fully stable, released version that is ALWAYS maintained by fixing bugs, a long term support for more years……ect).
2）`Current`：尝鲜版（A testable version that enhances something newly with new things coming, but NOT SURE whether they will be removed or changed in the future）.

And this will easily be understood by Chinese people and choose which to download.

Reply:
@Maledong so what you have suggested make sense to me, but it can lead to more confusion rather than clarification, since LTS is a widely accepted in opensource community. People tends to use Ubuntu LTS all the time, even we don't introduce a good translation.

Reply:
Yes, that just depends on which aspect you are on:)
But according to the actual result, many of us will look for what `LTS` is before they know about it means when download. Even this is a forever meaning in open-source env. 
I only want to make most of us (no matter whether he/she programs or not). When download Nodejs, he/she will easily understand what it means
Thanks anyway! Just a mention.